guides_and_processes/engineering_meetings.md

```diff
@@ -0,0 +1,266 @@
+  - [Meetings](#meetings)
+  - [Summary](#summary)
+  - [Standup (Daily, ~15 min)](#standup-daily-15-min)
+    - [Notes for engineers](#notes-for-engineers)
+  - [Sprint Demo (End of Sprint)](#sprint-demo-end-of-sprint)
+  - [Sprint Retro (End of Sprint)](#sprint-retro-end-of-sprint)
+  - [Sprint Planning (Start of Sprint)](#sprint-planning-start-of-sprint)
+  - [Backlog Grooming](#backlog-grooming)
+  - [Tech Discussion](#tech-discussion)
+  - [Catalyst Team Weekly Meeting](#catalyst-team-weekly-meeting)
+
+
+### Summary
+
+Meeting matrix for roles, note this is just a high level guide, attendance at meetings is determined by the role of each engineer.
+
+| Meeting                       | Architects         | Product Owner        | Eng. Manager         | Backend Developers            | Frontend Developers           | Designers                           | Design Manager                      | Testers/QA                         | SRE             | Operations       |
+|-------------------------------|--------------------|----------------------|----------------------|-------------------------|-------------------------|--------------------------------------|-------------------------------------|-------------------------------------|-----------------|-----------------|
+| Standup (Mon-Thu)             | X                  | (Optional)           | X                    | X                       | X                       | (Optional)                           | (Optional)                          | (Optional)                          | (Optional)      | (Optional)      |
+| Sprint Demo (End of Sprint)   | X                  | X                    | X                    | X                       | X                       | (Recommended)                        | (Recommended)                       | (Recommended)                       | (Recommended)    | (Recommended)    |
+| Sprint Retro (End of Sprint)  | X                  | X                    | X                    | X                       | X                       | X                                    | X                                   | X                                   | (As Required)    | (As Required)    |
+| Sprint Planning (Start)       | X                  | X                    | X                    | X                       | X                       | X          | X                        | X             | (As Required)      | (As Required)      |
+| Backlog Grooming              | X                  | X                    | X                    | (On Request)| (On Request)| (On Request)             | X           | (On Request)            | (On Request) | (On Request) |
+| Tech Discussion (Mon & Thu)   | (Optional)         | (Optional)           | (Optional)           | (Optional)              | (Optional)              | (Optional)                | (Optional)               | (Optional)               | (Optional) | (Optional) |
```

Comment:
Suggest changing this to `(twice weekly)` so we don't have to update the doc everytime we adjust the calendar.
Change `(Optional)` to `(Recommended)` for everyone.